content/docs/2.0/scalers/azure-storage-queue.md

```diff
@@ -17,10 +17,10 @@ triggers:
   metadata:
     queueName: functionsqueue
     queueLength: '5' # Optional. Queue length target for HPA. Default: 5 messages
-    connection: STORAGE_CONNECTIONSTRING_ENV_NAME
+    connectionFromEnv: STORAGE_CONNECTIONSTRING_ENV_NAME
 ```
 
-The `connection` value is the name of the environment variable your deployment uses to get the connection string. This is usually resolved from a `Secret V1` or a `ConfigMap V1` collections. `env` and `envFrom` are both supported.
+The `connectionFromEnv` value is the name of the environment variable your deployment uses to get the connection string. This is usually resolved from a `Secret V1` or a `ConfigMap V1` collections. `env` and `envFrom` are both supported.
 
 ### Authentication Parameters
```

Comment:
The connection parameter here is not in line with the sample, which one is correct?

Reply:
mentioned this in https://github.com/kedacore/keda-docs/pull/241/files#r486098364